plugin/src/test/groovy/org/gradle/testretry/SpockFuncTest.groovy

```diff
@@ -123,6 +123,16 @@ class SpockFuncTest extends AbstractTestFrameworkPluginFuncTest {
                     param << ['foo', 'bar', 'baz']
                     result << [true, false, true]
                 }
+
+                @spock.lang.Unroll
+                def "unrolled with param [#param]"() {
```

Comment:
Please add a separate test for the escaping and create a method that includes all Regex chars such that we are sure that we properly escape all of these chars.

Reply:
added a dedicated regex test